### ISSUE TITLE
Error in Angular 8 dynamic import

### ISSUE THREAD

**Issue by s-mizoguchi** [nestjs/ng-universal#101]
Transitions and errors occur on pages lazily loaded with dinamyc import.

```
ERROR { Error: Uncaught (in promise): TypeError: Cannot read property 'call' of undefined
TypeError: Cannot read property 'call' of undefined
    at __webpack_require__ (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:145323:30)
    at Function.requireEnsure [as e] (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:145342:25)
    at ɵ0 (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:145537:38)
    at RouterConfigLoader.loadModuleFactory (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:149667:39)
    at RouterConfigLoader.load (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:149652:35)
    at MergeMapSubscriber.project (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:148655:47)
    at MergeMapSubscriber._tryNext (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:35790:27)
    at MergeMapSubscriber._next (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:35780:18)
    at MergeMapSubscriber.Subscriber.next (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:32339:18)
    at Observable._subscribe (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:34351:24)
    at resolvePromise (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45498:31)
    at resolvePromise (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45455:17)
    at /Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45559:17
    at ZoneDelegate.invokeTask (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45069:31)
    at Object.onInvokeTask (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:27526:33)
    at ZoneDelegate.invokeTask (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45068:60)
    at Zone.runTask (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:44841:47)
    at drainMicroTaskQueue (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45247:35)
    at ZoneTask.invokeTask (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45148:21)
    at Server.ZoneTask.invoke (/Users/s-mizoguchi/Projects/universal-nest/dist/server.js:45133:48)
```

Update of Angular 8 looks inadequate #66

Is this upgrade guide compatible?

https://github.com/angular/universal/blob/master/docs/v8-upgrade-guide.md

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by kamilmysliwiec** [comment]
I'll take a look at this shortly

**Comment by s-mizoguchi** [comment]
Thank you for your quick reply.

This will reproduce.
```bash
$ git clone https://github.com/kamilmysliwiec/universal-nest.git
$ cd universal-nest && npm i
$ npm run build:ssr && npm run serve:ssr

# An error occurs when transitioning to the speakers.
```

Due to this problem, I can not set dynamic title and meta on the server side.

```typescript
  constructor(private title: Title, private meta: Meta) { }

  ngOnInit() {
    this.title.setTitle('Some Title');
    this.meta.addTag({ name: 'description', content: 'Some description' });
  }
```

**Comment by kamilmysliwiec** [comment]
Fixed in `@nestjs/ng-universal@2.0.0`. I have also updated universal-starter repo :) For existing apps, update all packages + follow this:
https://github.com/angular/universal/blob/master/docs/v8-upgrade-guide.md (only 1 bullet) + remove `enableProdMode()` from `main.ts`
